src/content/docs/dropins/all/extend-or-create.mdx

```diff
@@ -0,0 +1,193 @@
+   - No existing drop-in covers your use case
+   - You understand the maintenance responsibilities
+   
+   **If yes** → **Create** (early access, high risk)
+</Steps>
+
+<Aside type="tip">
+**Default recommendation**: Start with extending. Most customization needs can be met through slots, styling, and events without the complexity and maintenance burden of substitution or creation.
+</Aside>
+
+## The three approaches
+
+### 1. Extend (Recommended)
+
+This is the **default and fully supported** approach. Extending allows you to customize drop-ins while maintaining compatibility and receiving updates.
+
+**When to extend:**
+- You need to add custom content or UI elements
+- You want to change styling or layouts
+- You need custom business logic that responds to drop-in events
+- You want to modify how data is displayed or processed
+
+**Extension methods:**
+- **Slots** - Inject custom HTML/components at predefined points
+- **Styling** - Override CSS, modify layouts, replace components
+- **Events** - Listen to data events and add custom behavior
+- **Configuration** - Modify drop-in settings and options
+- **Transformers** - Change how drop-ins process and display data
+
+**Benefits:**
+- Fully supported by Adobe
+- Automatic compatibility with updates
+- Lower maintenance overhead
+- Access to new features and bug fixes
+
+### 2. Substitute (Proceed with caution)
+
+Replace an Adobe drop-in entirely with a third-party solution. This is common in practice but comes with significant responsibilities.
+
+**When substitution happens:**
+- Checkout flows (Stripe, PayPal, Bold Commerce, etc.)
+- Payment processing with specific provider requirements
+- Specialized functionality that doesn't align with Adobe's drop-in approach
+- Legacy system integration requirements
+
+**Risks and responsibilities:**
+- **Maintenance burden** - You own all updates, bug fixes, and compatibility
+- **API changes** - Must adapt to Commerce API changes independently
+- **Feature gaps** - May miss out on new Commerce features
+- **Support limitations** - Adobe cannot provide support for third-party code
+
+<Aside type="caution">
+**Important**: Substitution means you're responsible for maintaining compatibility with Commerce APIs and handling all future updates yourself.
+</Aside>
+
+### 3. Create (Early access - experimental)
+
+Build entirely new drop-ins from scratch using the SDK. This is currently in early access with limited support.
+
+**When creation might be considered:**
+- Building reusable components across multiple storefronts or brands
+- No existing drop-in addresses your specific use case
+- You have significant development resources and expertise
+- You understand the long-term maintenance commitment
+
+**Current limitations:**
+
+<Aside type="danger">
+**Early Access Warning**: The drop-in SDK is currently experimental and unsupported for third parties. Building new drop-ins comes with significant risks:
```

Comment:
This wording is a bit strong for me. It makes it sound like the SDK is experimental, but I know what you are trying to convey - the use of the SDK (outside of Adobe) is "EA".

Reply:
We don't want to position it as drop-ins are experimental and not stable.

I think a "Warning" is enough. We just want to convey that it's "Early Access" with limited support for third parties. That's it.